components/servo/Cargo.toml

```diff
@@ -32,6 +32,7 @@ webgl_backtrace = [
 ]
 
 [dependencies]
+aes = { workspace = true }
```

Comment:
Why is this added here?

Reply:
Ah my bad, must be left over from when I was still figuring things out, will remove.